ironic/drivers/modules/drac/raid.py

```diff
@@ -644,8 +724,12 @@ def _commit_to_controllers(node, controllers):
         return
 
     driver_internal_info = node.driver_internal_info
+    driver_internal_info['raid_config_substep'] = substep
+
     if 'raid_config_job_ids' not in driver_internal_info:
         driver_internal_info['raid_config_job_ids'] = []
+    if 'raid_config_parameters' not in driver_internal_info:
```

Comment:
I think we should unconditionally set the raid_config_parameters to [].  This will make it so that we are guaranteed that there is no old data in it.

Reply:
Done